doc/Hacking.md

```diff
@@ -1,6 +1,23 @@
 Hacking
 =======
 
+## How to Debug
+
+We have a helper function called `logger.dbg` to help with debugging. You can use that function to print string and tables:
+
+```lua
+local logger = require("logger")
+a = {"1", "2", "3"}
+logger.dbg("table a: ", a)
+```
+
+Anything printed by `logger.dbg` starts with a `#` sign.
+
+## Bug hunting in kpv
+
+A real example for hunting bug in KPV's cache system: https://github.com/koreader/kindlepdfviewer/pull/475
```

Comment:
I barely even read it diagonally after making the link to point to the right repository, but it looked like the comments around https://github.com/koreader/kindlepdfviewer/pull/475#issuecomment-9637423 might still be somewhat relevant to #2502. I'd never even heard of KOReader at the time (or kindlepdfviewer) and you added it to the wiki, so it's up to you. :-)